BinEncoder.py

```diff
@@ -0,0 +1,49 @@
+from enum import Enum
+
+from MessagersInterfaces import Listener, Notifier, T
+from RossServer import *
+from UMDDecoder import *
+
+
+class RossEventToBin(Listener[RossEvent], Notifier[bytes]):
+    def __init__(self, listener: Listener[bytes], num_of_cam=127):
+        '''
+        Creates "bytes" with lenght of 33
+        1-st byte - lenght of "bytes"
+        Others - camera's states
```

Comment:
after that each block of 2 bits represents one camera, where first bit is boolean PGM state, second - PVW state